### ISSUE TITLE
Problem with resizing images

### ISSUE THREAD

**Issue by madoar** [PlayOnLinux/POL-POM-5#757]
When opening POL 5, with the default scaling (the slider in the settings is in the middle), POL 5 will look like this:
![grafik](https://cloud.githubusercontent.com/assets/18488086/25745220/21d8d19e-319e-11e7-835c-5a64ac09e879.png)
We need to make sure that the images are resized too, and that the text below the images are fully visible.

If you change the slider to the left, the text below the images will disappear.

**Comment by plata** [comment]
Agree to scaling the pictures. However, the broken text is not related to that. I will update the themes, that should fix it.

**Comment by plata** [comment]
Regarding the images: Do you have an idea how to do this? I've tried that in #746 already but it's not so straight forward.

**Comment by madoar** [comment]
Sorry I have no clue too. 
As far as I understand it the font size should apply to images too, but in some sporadic cases it doesn't work.
If we can't solve this we should think about using another approach to scaling/resizing the themes.

**Comment by plata** [comment]
The problem here is that the image is created from a byte[] inside the code. It should work if we would set the path instead and define the size in the CSS.

**Comment by madoar** [comment]
So we need to create a css file on the fly, for the images?
It would be nice if we could do this semi persistent (i.e. not in a temp folder), what do you think?

**Comment by plata** [comment]
Don't think that's needed. We can directly set the CSS code for the image in the code.
```Java
xy.setStyle("-fx-background-image: url('...');");
```

**Comment by madoar** [comment]
You're right this is more elegant and doesn't need us to create a temp file at all.

**Comment by madoar** [comment]
Should we create a new issue for the extraction of the image data to the style properties, or should we do that in the scope of this Issue too?

**Comment by plata** [comment]
I guess the question is obsolete after #773, right?

**Comment by madoar** [comment]
Yes, it should be.
